openbas-api/src/main/java/io/openbas/rest/tag_rule/TagRuleApi.java

```diff
@@ -58,7 +70,8 @@ public List<TagRuleOutput> tags() {
         @ApiResponse(responseCode = "200", description = "TagRule deleted"),
```

Comment:
204 instead of 200 ?

Reply:
Like others, we really do send a 204 back.